### DISCUSSION TITLE
How to install magma after using nvidia wheels

### DISCUSSION THREAD

**Discussion by Madelynnn** [kornia/kornia#2755] [Questions]
### Describe the bug

The pytorch wheels provided by nvidia jetson come without MAGMA. Hence, the function get_perspective_transform which uses torch.linalg.solve will throw the error: "RuntimeError: solve: MAGMA library not found in compilation. Please rebuild with MAGMA".

How can this error be avoided?

Thanks in advance for answering my question

### Reproduction steps

```bash
1.
2.
3.
...
```

### Expected behavior

Well, that it should work fine or that the error gives a solution on how to solve it while using the nvidia wheels. 

### Environment

```shell
torch 1.9.0
torchvision 0.10.0
kornia 0.6.7
```

**Comment by edgarriba** [comment]
I had the same issue few months back. First i computed that specific augmentation in cpu, later we compiled pytorch with magma. I also found that using a different algebra solver worked but the results were not stable. TBH, nvidia wheels for jetson are a bit outdated and I don’t see any progress in that direction. The best is to compile pytorch with your own specs. We could discuss to open source our internal wheels at the company I work also to help kornia community /cc @ethanrublee
